server/graphql/requisition/src/mutations/update_indicator_value.rs

```diff
@@ -0,0 +1,90 @@
+use async_graphql::*;
+use graphql_core::standard_graphql_error::StandardGraphqlError;
+use graphql_core::{
+    simple_generic_errors::RecordNotFound, standard_graphql_error::validate_auth, ContextExt,
+};
+use graphql_types::types::program_indicator::IndicatorValueNode;
+use service::auth::{Resource, ResourceAccessRequest};
+use service::requisition::indicator_value::{UpdateIndicatorValue, UpdateIndicatorValueError};
+
+#[derive(InputObject)]
+pub struct UpdateIndicatorValueInput {
+    pub id: String,
+    pub value: String,
```

Comment:
Carryover issue for Requisition checks. Ok to leave for now and just have frontend block